js/app.js

```diff
@@ -51,13 +72,17 @@
    * using the Function prototype method: apply
    * to set the context to the correct object
    */
-
+btn2.addEventListener('click', () => {
+  setMyLightGreen.apply(btn2);
+})
 
   /*
    * Add a click event listener to btn3
    * the handler method will be a reference to the setMyLightGreen function
    */
-
+btn3.addEventListener('click', () => {
```

Comment:
The instructions ask to set the handler method to a reference to the setMyLightGreen function. It does not ask to create a anonymous function that ends up calling setMyLightGreen